app/tests/system/setup/setup.js

```diff
@@ -0,0 +1,142 @@
+/*
+ * Loop through the custom contexts that exist when the app starts
+ * and grab their names.
+ *
+ * To change the environment we're testing against, specify it in site.json.
+ * This should align with what the app is built against, which is declared in
+ * app/environment.js hostUrl
+ */
+
+var Site = require('../site.json');
+
+var Setup = {};
+
+var selectors = {
+    previewButton: '#authorize',
+    registeredAuctionList: '.js-auction-lists'
+};
+
+var expected = {
+    //EXAMPLE Webviews constructed for project.
+    'noInternet': /no-internet\.html/i,
+    'couldNotLoad': /could-not-load\.html/i
+};
+
+Setup.appPreview = function(browser) {
+    browser.contexts(function(result) {
+        browser
+            .session(function(type) {
+                browser
+                    .log('Running on platform - ' + type.value.platformName);
+                if (type.value.platformName === 'Android') {
```

Comment:
Currently in some situation, it is "platform", but sometimes it is "platformName", and the value is "ANDROID" or "Android", and I haven't figure out the pattern. If later we figure out the pattern, we could create a new PR for it.